pcsx2/gui/MainFrame.cpp

```diff
@@ -665,7 +665,7 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	menubar.Enable( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
+	menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
```

Comment:
I guess it's possible, (looking at the [Wx Reference](http://docs.wxwidgets.org/trunk/classwx_menu_bar.html#ab53b19f2e8b0e7b39ae551d51f2f6f6b) ) but then why doesn't this function correctly with enable used. I feel like this worked at one point, but I can't recall